spirv_msl.hpp

```diff
@@ -789,6 +852,7 @@ class CompilerMSL : public CompilerGLSL
 
 	std::unordered_set<uint32_t> buffers_requiring_array_length;
 	SmallVector<uint32_t> buffer_arrays;
+    std::set<SPIRVariable *> atomic_vars; // Emulate texture2D atomic operations
```

Comment:
Does this have to be a set and not unordered_set?

Reply:
Thanks for pointing that out. This set is only used to check if a given key exists, so it can also be an unordered set.